src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -111,6 +111,27 @@ def migrate(self, daemon_name=None, realm_name=None, zonegroup_name=None, zone_n
                                                          secret_key)
         return result
 
+    @RESTController.Collection(method='PUT', path='/setup_multisite_replication')
```

Comment:
this sounds like a POST action to me and we can simply call the endpoint `POST /multisite-replications` rather than `setup_multisite_replication`. And as per the REST API standards, its better to use hyphens to separate the words instead of underscores, which will improve readability and easability.  
```suggestion
    @RESTController.Collection(method='PUT', path='/setup_multisite_replication')
```